src/views/layouts/default.njk

```diff
@@ -1,5 +1,22 @@
+{% set sections = [
+    {
+        pages: [
+            {
+                url: "/",
+                title: "Getting started"
+            }
+        ]
+    }, {
+        title: "Styles",
+        pages: collections.styles | eleventyNavigation
+    }, {
+        title: "Components",
+        pages: collections.components | eleventyNavigation
+    }
+] %}
+
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="text-grey-1">
     <head>
         <meta charset="utf-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Not part of this PR but I haven't looked at the other PRs until this point. Not a requirement for this PR but the meta tag 
```html
<meta description="{{ description }}">
```
seems strange to me.
Shouldn't it be something like
```html
<meta name="description" content="{{ description }}"
```
Unless I am missing the use case of it.

Reply:
Yep, you're right—well spotted. I'll change it here.